druid/src/core.rs

```diff
@@ -411,6 +418,14 @@ impl<T: Data, W: Widget<T>> WidgetPod<T, W> {
             return;
         }
 
+        if !self.is_initialized() {
+            debug_panic!(
+                "{:?} is supposed to paint without having first received WidgetAdded.",
```

Comment:
I find this wording slightly awkward? might just say something like, paint method called before WidgetAdded."